applications/CoSimulationApplication/python_scripts/coupling_operations/distribute_point_values.py

```diff
@@ -1,38 +1,65 @@
 from __future__ import print_function, absolute_import, division  # makes these scripts backward compatible with python 2.6 and 2.7
 
+# Importing the Kratos Library
+import KratosMultiphysics as KM
+
 # Importing the base class
 from KratosMultiphysics.CoSimulationApplication.base_classes.co_simulation_coupling_operation import CoSimulationCouplingOperation
 
 class DistributePointValuesOperation(CoSimulationCouplingOperation):
-    # this is a dummy implementation, this should use the "VariableRedistributionUtility" from the core
-    def Initialize(self):
-        pass
-
-    def Finalize(self):
-        pass
-
-
-    def InitializeSolutionStep(self):
-        pass
 
-    def FinalizeSolutionStep(self):
-        pass
+    def __init__(self, settings, solver_wrappers):
+        super(ComputeNormalsOperation, self).__init__(settings)
+        solver_name = self.settings["solver"].GetString()
+        data_name = self.settings["data_name"].GetString()
+        self.interface_data = solver_wrappers[solver_name].GetInterfaceData(data_name)
 
+        self.redistribution_iterations = self.settings["redistribution_tolerance"].GetDouble()
+        self.redistribution_tolerance  = self.settings["redistribution_iterations"].GetInt()
 
-    def InitializeCouplingIteration(self):
-        pass
+        if self.interface_data.is_scalar_variable:
+            self.intermediate_variable = KM.KratosGlobals.GetVariable(self.settings["intermediate_variable_scalar"].GetString())
+        else:
+            self.intermediate_variable = KM.KratosGlobals.GetVariable(self.settings["intermediate_variable_scalar"].GetString())
 
-    def FinalizeCouplingIteration(self):
-        pass
+    def Execute(self):
+        KM.VariableRedistributionUtility.DistributePointValues(
+            self.interface_data.GetModelPart(),
+            self.intermediate_variable,
+            self.interface_data.variable,
+            self.redistribution_tolerance,
+            self.redistribution_iterations)
```

Comment:
I think the order of the 2nd and the 3rd arguments of this function should be changed as well.
for example in a Dirichlet-Neumann coupling, this function would be used on the fluid's FSI interface to convert TRACTION as interface_data.variable to "distributed tractions" in an intermediate auxiliary variable.